package.json

```diff
@@ -20,16 +20,20 @@
     "build": "react-scripts build",
     "clean": "rm -rf ./node_modules/.cache/eslint-loader",
     "docs": "typedoc --options typedoc.json",
-    "deploy": "npm version major && npm run build && firebase deploy",
+    "deploy": "npm version major && npm run build && firebase deploy --only hosting",
+    "deploy:functions": "cat ./firebase-config.json | node ./scripts/setFirebaseConfig.js && npm version major && cd ./functions && npm run build && firebase deploy --only functions",
```

Comment:
Remove `npm version major` or move it to after the `cd` into `functions/`. We don't want to bump the core project version when functions are deployed.